plugins/modules/make.py

```diff
@@ -192,6 +192,7 @@ def main():
         chdir=module.params['chdir'],
         file=module.params['file'],
         jobs=module.params['jobs'],
+        command=' '.join(base_command),
```

Comment:
I wonder whether this one should be correctly quoted.

With `from ansible.module_utils.six.moves import shlex_quote` you can do `command=' '.join([shlex_quote(part) for part of base_command])`.

Reply:
hmmmm this means we should also quote it when we call the command - unless it's done within the run_command ? I have to check that out. Pretty sure it does the necessary string sanitizing.

Reply:
Seems to be properly escaped yep:
https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/basic.py#L1926

I'll push an update addressing both your comments once the doc patch is in, so that I can rebase and add the new entry in the RETURN part.